utils/record_training.py

```diff
@@ -0,0 +1,161 @@
+import argparse
+import os
+import re
+import shutil
+import subprocess
+from copy import deepcopy
+
+from utils.utils import ALGOS, get_latest_run_id
+
+if __name__ == "__main__":  # noqa: C901
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--env", help="environment ID", type=str, default="CartPole-v1")
+    parser.add_argument("-f", "--folder", help="Log folder", type=str, default="rl-trained-agents")
+    parser.add_argument("-o", "--output-folder", help="Output folder", type=str)
+    parser.add_argument("--algo", help="RL Algorithm", default="ppo", type=str, required=False, choices=list(ALGOS.keys()))
+    parser.add_argument("-n", "--n-timesteps", help="number of timesteps", default=1000, type=int)
+    parser.add_argument("--n-envs", help="number of environments", default=1, type=int)
+    parser.add_argument("--deterministic", action="store_true", default=False, help="Use deterministic actions")
+    parser.add_argument("-g", "--gif", action="store_true", default=False, help="Convert final video to gif")
+    parser.add_argument("--seed", help="Random generator seed", type=int, default=0)
+    parser.add_argument(
+        "--no-render", action="store_true", default=False, help="Do not render the environment (useful for tests)"
```

Comment:
this argument is not used, is it intended?

Reply:
Only in case it is somehow needed when calling `record_video.py` (for testing purposes perhaps?), but I forgot to use it there. From a feature point of view, I don't intend to use it.

Reply:
ok, then please remove it and we are good to go ;)